.gitignore

```diff
@@ -57,3 +57,7 @@ gemfiles/*
 
 # bundle config
 gemfiles/.bundle
+
+# RubyMine/IntelliJ
+.idea
+*.iml
```

Comment:
I'd suggest you add these IDE-specific ignore rules under your `~/.gitignore` instead, and configure git to use it like so: `git config --global core.excludesfile '~/.gitignore'`.